docs/quickstart/quickstart_chains/cosmos-agoric.md

```diff
@@ -0,0 +1,344 @@
+# Agoric Quick Start
+
+## Goals
+
+The goal of this quick start guide is to index all transfer events and messages on the [Agoric network](https://agoric.com/) 
+
+::: info
+The Agoric network is a chain based on the Cosmos SDK, which means you can index chain data via the standard Cosmos RPC interface.
+
+Before we begin, make sure that you have initialised your project using the provided steps in the **[Start Here](../quickstart.md)** section. You must complete the suggested [4 steps](https://github.com/subquery/cosmos-subql-starter#readme) for Cosmos users.
+:::
+
+In every SubQuery project, there are 3 key files to update. Let's begin updating them one by one.
+
+::: tip
+The final code of this project can be found [here](https://github.com/subquery/cosmos-subql-starter/tree/main/Agoric/agoric-starter).
+:::
+
+## 1. Your Project Manifest File
+
+The Project Manifest (`project.ts`) file is an entry point to your project. It defines most of the details on how SubQuery will index and transform the chain data. For Cosmos chains, there are four types of mapping handlers (and you can have more than one in each project):
+
+- [BlockHanders](../../build/manifest/cosmos.md#mapping-handlers-and-filters): On each and every block, run a mapping function
+- [TransactionHandlers](../../build/manifest/cosmos.md#mapping-handlers-and-filters): On each and every transaction, run a mapping function
+- [MessageHandlers](../../build/manifest/cosmos.md#mapping-handlers-and-filters): On each and every message that matches optional filter criteria, run a mapping function
+- [EventHanders](../../build/manifest/cosmos.md#mapping-handlers-and-filters): On each and every event that matches optional filter criteria, run a mapping function
+
+Note that the manifest file has already been set up correctly and doesn’t require significant changes, but you need to change the datasource handlers. This section lists the triggers that the manifest file looks for on the blockchain to start indexing.
+
+```ts
+{
+  dataSources: [
+    {
+      kind: CosmosDatasourceKind.Runtime,
+      startBlock: 11628269,
+      mapping: {
+        file: "./dist/index.js",
+        handlers: [
+          // {
+          //     Using block handlers slows your project down as they can be executed with each and every block.
+          //     Only use if you need to
+          //     handler: 'handleEvent',
+          //     kind: CosmosHandlerKind.Block,
+          // },
```

Comment:
Please remove this commented section for brevity

Reply:
Done.